docs/kafka/getting-started-kafka/README.adoc

```diff
@@ -138,7 +138,7 @@ endif::[]
 . On the *Kafka Instances* page of the {product-kafka} {service-url-kafka}[web console^], click *Create Kafka instance*.
 . Complete the *Create a Kafka instance* form to define the following instance details. Some instance details currently have only one option.
 * *Instance name*: Enter a unique name for the Kafka instance, such as `my-first-kafka-instance`.
-* *Cloud provider*: For a production Kafka instance, accept the default value of *Amazon Web Services*. If you're creating a trial instance, you also have the option to select *Google Cloud Platform*.
+* *Cloud provider*: For a production Kafka instance, accept the default value of *Amazon Web Services*. 
```

Comment:
If there are no Prod versus trial now, do we even need to mention "production", should we word it the way "Availability zones" are? 
Should we say "The selection for cloud provider is preconfigured as *Amazon Web Services* for a production instance"